website/content/en/status/report-2023-01-2023-03/openstack-on-freebsd.adoc

```diff
@@ -0,0 +1,37 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+This project aims to port key OpenStack components so that FreeBSD can function as an OpenStack host.
+
+In 2023 Q1, the big news is that we're able to spawn FreeBSD instances with `bhyve` on the OpenStack platform.
```

Comment:
Could you please use the man macro here: `man:bhyve[8]`.

Reply:
Updated and please take a look. Thank you.